ISISComputingGroup/EPICS-inst_servers
# Ticket1446: Don't push configs onto the wrong branch

### Description of work

See https://github.com/ISISComputingGroup/IBEX/issues/1446

Version control is only started when on the same branch as your host name 
### To test
- Start an instrument with your configurations under git version control, confirm that any modifications/additions made through the blockserver are added to the remote repo
- Stop the instrument, switch your configurations to another branch
- Start the instrument and confirm that no config changes are pushed automatically when modifications are made through the blockserver

---
#### Code Review
### Functional Tests
### Final steps
